activemq-console/src/main/java/org/apache/activemq/console/command/store/StoreBackup.java

```diff
@@ -88,6 +96,14 @@ public void execute() throws Exception {
             throw new Exception("optional --offset and --count must be specified together");
         }
 
+        if ((startMsgId != null || endMsgId != null) && queue == null) {
+            throw new Exception("optional --queue must be specified when using startMsgId or endMsgId");
```

Comment:
All these exceptions should be more specific, such as IllegalArgumentException.

All the checks here would benefit from something like [Preconditions](https://guava.dev/releases/snapshot/api/docs/com/google/common/base/Preconditions.html) and the checkArgument() methods if we can use guava.